cinch/roles/jenkins_slave/templates/swarm.upstart.conf

```diff
@@ -0,0 +1,21 @@
+description "Jenkins slave swarm client"
```

Comment:
Something I missed here the first time around is that this template file does not end with a .j2 extension.  Is this a standard convention we should be following?